src/components/CreateAccount.vue

```diff
@@ -41,19 +41,7 @@
               <p class="mb-3">
                 {{
                   $gettext(
-                    "Please enter a password (and confirm it) " +
-                      "for your new wallet %{ backend }.",
-                    {
-                      backend: form.accountBackend,
-                    }
-                  )
-                }}
-              </p>
-              <p class="mb-3">
-                {{
-                  $gettext(
-                    "This password is different from your login password, " +
-                      "it protects your wallet %{ backend }.",
+                    "Create the password for your wallet. (It is different from your user account password and is only related to your wallet)",
```

Comment:
why parens ? Why change the previous phrasing ?

Reply:
it was requested in this: https://lokavaluto.fr/web#id=1482&action=259&active_id=7&model=project.task&view_type=form&cids=1&menu_id=144

Reply:
no parens in instructions. several typos. Corrected and pushed.